server/modules/email/emailModule.js

```diff
@@ -41,6 +42,16 @@ const emailTransport = nodemailer.createTransport({
   },
 });
 
+
+const sendMail = async (mailOptions) => {
+  try {
+    await emailTransport.sendMail(mailOptions);
+    return { status: 'success' };
+  } catch (error) {
+    return { status: 'error', message: error.message };
+  }
```

Comment:
A helper function has been created to aid the sending of response(either error or success), please make use of it here for sending both success and error response.
You can check `server/helper/response.js` to see how to use it.

Reply:
implemented this